#This message allows the robot to send it's core information
pose robot_pose
velocity robot_velocity
pose ball_position
velocity ball_velocity
bool has_ball
bool sees_ball
obstacle[] obstacles
interestPoint[] interest_points
